.github/workflows/official_release.yml

```diff
@@ -0,0 +1,65 @@
+
+
+# Beware this is a dangerous script 👻
+name: Make an official Release
+
+# Requires manual trigger
+on: workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+    name: Test changed-files
+    steps:
+      - uses: trstringer/manual-approval@v1
+        with:
+          secret: ${{ github.TOKEN }}
+          approvers: msaroufim,lxning
+          minimum-approvals: 2
+          issue-title: "Offical Release"
+      - name: Setup Conda
+        uses: s-weigand/setup-conda@v1
+      - run: conda --version
+      - run: conda install -y conda-build anaconda-client
+      - name: Checkout TorchServe
+        uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          python ts_scripts/install_dependencies.py --environment=dev
+          pip install -e .
+      - name: Setup Java 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - name: Build PyPI & conda nightly binaries
+        run: python binaries/build.py
+
+      - name: Push conda nightly binaries
+        env:
+          ANACONDA_API_TOKEN: ${{ secrets.CONDA_NIGHTLY_TOKEN }}
+        run: python binaries/upload.py --upload-conda-packages
+      - name: Push PyPI binaries
+        env:
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+        if: matrix.os == 'ubuntu-latest'
+        run: python binaries/upload.py --upload-pypi-packages
+      - name: Login to Docker
+        env:
+          DOCKER_PASSWORD: ${{secrets.DOCKER_PASSWORD}}
+          run: docker login --username pytorchbot --password "$DOCKER_PASSWORD"
+      - name: Build & Upload pytorch/torchserve Docker images
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cd docker
+          python build_upload_release.py
+          cd ..
+      - name: Build & Upload pytorch/torchserve-kfs Docker images
+        if: matrix.os == 'ubuntu-latest'
+        run: |
+          cd kubernetes/kserve
+          python build_upload_release.py
```

Comment:
why can't we do a python kubernetes/kserve/build_upload_release.py

Reply:
https://github.com/pytorch/serve/pull/1739#discussion_r921708896